src/java.base/share/classes/java/lang/System.java

```diff
@@ -2244,6 +2244,11 @@ private static void initPhase3() {
         // SecurityManager
         Unsafe.getUnsafe().ensureClassInitialized(StringConcatFactory.class);
 
+        // Emit a warning if java.io.tmpdir is set via the command line to a directory that doesn't exist
```

Comment:
could you split this comment across 2 lines ? - Keeps length similar to comment just above this code.